src/PluginSession.php

```diff
@@ -167,12 +199,12 @@ protected function closeSession() {
 	 * (DEPRECATED) Translate a base64 string to PEM encoded public key.
 	 *
 	 * @param string $data base64 encoded key
-	 *
+	 * @deprecated
 	 * @return string PEM encoded key
 	 */
 	public static function base64ToPEMPublicKey($data) {
 
-		error_log("Warning: PluginSession::base64ToPEMPublicKey() is deprecated. Please switch over to  SSOToken::base64ToPEMPublicKey().");
+		error_log("Warning: PluginSession::base64ToPEMPublicKey() is deprecated. Please switch over to	SSOToken::base64ToPEMPublicKey().");
```

Comment:
just a suggestion..
instead of using error_log() (without the second and rest arguments this function logs the message in a error file)
I suggest to use [trigger_error()](https://www.php.net/manual/en/function.trigger-error.php) because  you can use set_error_handler() to handle these.